etc/config/ocaml.defaults.properties

```diff
@@ -1,9 +1,7 @@
-compilers=/usr/bin/ocamlopt
```

Comment:
Yeah, it's overriden, but just when running from the proper environment. Local users that _just_ run the site will have not created any .local file for ocaml, but it's nice that they still see a compiler if they happen to have it installed without further modifications needed

Reply:
one more thing, should I use an absolute path? Consider that many modern OCaml setups are provided as opam switches (the language's package manager) which set a path in the local shell's environment.. so if executables can be called without an absolute path, this may be preferable.

Reply:
I might need to double-check, but I think it can be called without an absolute path yeah

Reply:
Yeah, it's possible. Python already does this https://github.com/compiler-explorer/compiler-explorer/blob/main/etc/config/python.defaults.properties